src/shared/services/bolt/boltMappings.test.js

```diff
@@ -367,4 +368,221 @@ describe('boltMappings', () => {
       expect(out.nodes.length).toEqual(4)
     })
   })
+  describe('applyGraphTypes', () => {
+    test('should apply integer type', () => {
+      const rawNumber = { low: 5, high: 0 }
+      const typedNumber = applyGraphTypes(rawNumber)
+      expect(typedNumber instanceof neo4j.Integer).toEqual(true)
+    })
+
+    test('should apply node type', () => {
+      const rawNode = {
+        labels: ['Test'],
+        properties: [],
```

Comment:
Please add some properties of different types (integers, string, array, null, object) to make sure they come along.